### DISCUSSION TITLE
is it possible to increase size of the emote autocomplete list?

### DISCUSSION THREAD

**Discussion by Brog33** [Chatterino/chatterino2#3750] [Q&A]
I can barely make out the emote autocomplete. The emotes shows up tiny compared to other UI elements - and it doesn't follow the emote size option in the settings. Any other who experience this?

![image](https://user-images.githubusercontent.com/1303537/169364589-db2e9204-ca14-46d6-8783-d0c836fa53df.png)

**Comment by pajlada** [comment]
This feels like a bug to me, it should resize to appropriately fill those elements. Created https://github.com/Chatterino/chatterino2/issues/3754 to track it.
